python/tests/test_options.py

```diff
@@ -135,58 +248,264 @@ def test_argument_error_from_dict(self):
+        assert options.strict_file_interactability is True
+        assert options.timeouts == timeouts
 
-        sauce.accept_insecure_certs = True
+    def test_sauce_options(self):
+        prerun = {'executable': 'http://url.to/your/executable.exe',
+                  'args': ['--silent', '-a', '-q'],
+                  'background': False,
+                  'timeout': 120}
+        custom_data = {'foo': 'foo',
+                       'bar': 'bar'}
+        tags = ['foo', 'bar']
+
+        options = SauceOptions()
+
+        options.avoid_proxy = True
+        options.build = 'Sample Build Name'
+        options.capture_performance = True
+        options.chromedriver_version = '71'
+        options.command_timeout = 2
+        options.custom_data = custom_data
+        options.extended_debugging = True
+        options.idle_timeout = 3
+        options.iedriver_version = '3.141.0'
+        options.max_duration = 300
+        options.name = 'Sample Test Name'
+        options.parent_tunnel = 'Mommy'
+        options.prerun = prerun
+        options.priority = 0
+        options.public = 'team'
+        options.record_logs = False
+        options.record_screenshots = False
+        options.record_video = False
+        options.screen_resolution = '10x10'
+        options.selenium_version = '3.141.59'
+        options.tags = tags
+        options.time_zone = 'San Francisco'
+        options.tunnel_identifier = 'tunnelname'
+        options.video_upload_on_pass = False
+
+        assert options.avoid_proxy is True
+        assert options.build == 'Sample Build Name'
+        assert options.capture_performance is True
+        assert options.chromedriver_version == '71'
+        assert options.command_timeout == 2
+        assert options.custom_data == custom_data
+        assert options.extended_debugging is True
+        assert options.idle_timeout == 3
+        assert options.iedriver_version == '3.141.0'
+        assert options.max_duration == 300
+        assert options.name == 'Sample Test Name'
+        assert options.parent_tunnel == 'Mommy'
+        assert options.prerun == prerun
+        assert options.priority == 0
+        assert options.public == 'team'
+        assert options.record_logs is False
+        assert options.record_screenshots is False
+        assert options.record_video is False
+        assert options.screen_resolution == '10x10'
+        assert options.selenium_version == '3.141.59'
+        assert options.tags == tags
+        assert options.time_zone == 'San Francisco'
+        assert options.tunnel_identifier == 'tunnelname'
+        assert options.video_upload_on_pass is False
 
-        assert sauce.browser_name == 'safari'
-        assert sauce.accept_insecure_certs is True
-        assert sauce.platform_name == 'macOS 10.14'
 
-    def test_sauce_options(self):
-        sauce = SauceOptions()
-        sauce.name = 'my-test'
-        sauce.record_screenshots = False
+class TestSauceSpecificOptions(object):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [redefinition of unused 'TestSauceSpecificOptions' from line 248](https://app.codacy.com/manual/SauceLabs/simple_sauce/pullRequest?prid=4773242)